packages/myst-frontmatter/src/contributors/authors.yml

```diff
@@ -134,6 +134,33 @@ cases:
             family: Name
           email: example@example.com
           corresponding: false
+  - title: No email is corresponding if first is false
```

Comment:
I noticed this behaviour and wanted to raise it:

Is it intentional that specifying `corresponding: false` on one author, prevents any other author from implicitly getting the `corresponding: true` flag?

I can see arguments either way:

(Yes, to the above question): the user knows about the `corresponding` flag, as they used it on one of their authors. If they did want a corresponding author, they would have known to add it.
(No, to the above question): we should always try to have a corresponding author where possible.

Reply:
Lets make the change! And update this test case.

Reply:
Updated!